### DISCUSSION TITLE
What is the best front end to use with vLLM?

### DISCUSSION THREAD

**Discussion by PaulConyngham** [vllm-project/vllm#347] [General]
Hi guys,

Over the weekend I experimented with getting [ChatbotUI](https://github.com/mckaywrigley/chatbot-ui) and [Fastchat](https://github.com/lm-sys/FastChat) to work with VLLM without much success. 

What is your current preferred go to Front end to pair with this project?

**Comment by gesanqiu** [comment]
Have you tried after this https://github.com/lm-sys/FastChat/pull/1835
